docs/operators/from_http.md

```diff
@@ -160,6 +160,12 @@ The maximum size of an incoming request to accept.
 
 Defaults to `10MiB`.
 
+### `max_connections = int (optional)`
+
+The maximum number of simultaneous incoming connections to accept.
+
+Defaults to `1000`.
```

Comment:
Maybe we should use a much lower default? With the current buffer setup, 1000 pipelines can easily OOM a node. Maybe 10 is more reasonable for now?